### ISSUE TITLE
Chat Staying open

### ISSUE THREAD

**Issue by tenroy6** [night/BetterTTV#1895]
The chat keeps welcoming me back when I manually hide it to full theatre mode. We need an option to keep it open so peoples chats continue there.

**Comment by gapbl4** [comment]
Why not watch stream in normal fullscreen instead?

**Comment by tenroy6** [comment]
Because I want to be able to go to chat when I want to without having to press escape? Plus I don't have the money to own a second monitor nor the space? Also I have about 10 other tabs going and playing a game myself while im watching the stream on the side.


Sent from Outlook<http://aka.ms/weboutlook>


________________________________

**Comment by gapbl4** [comment]
Have you tried disabling bttv? Doesn't seem like that problem is bttv extension related.

**Comment by tenroy6** [comment]
It works when I don't use bttv. I think it's a memory problem when having enable theatre mode automatically. As when I turn it off, no more lag. No more extreme problems with the player or chat. There's definitely a bug in that option.


Sent from Outlook<http://aka.ms/weboutlook>


________________________________

**Comment by gapbl4** [comment]
![2016-12-18_10-07-51](https://cloud.githubusercontent.com/assets/14076068/21292107/f2ac507c-c509-11e6-9c99-04d4a50cd137.gif) Just tried in incognito mode with disabled bttv. It still disconnects.

**Comment by tenroy6** [comment]
Yea, that's the feature I want gone. I want it to not do that so we don't open it and miss everyone that talked in between.

However I suppose this wasn't the bug form I submitted about loading correctly. The site doesn't load correctly when auto theatre mode is enabled. When I turn it off the site is fine and the chat ACTUALLY loads. When I have to enable I have to reload the page from 1-5 times just to actually be welcomed to the chat and people actually chat and what I saw actually pops up.



  Sent from Outlook
